### ISSUE TITLE
Feature Request: Scene Support

### ISSUE THREAD

**Issue by andreasnanko** [c727/home-assistant-tiles#9]
Hi,

thanks for this awesome piece of work.

I was wondering if it's possible to activate scenes, but it seems that it always calls homeassistant.turn_off. I did not find any reference to scenes in the code, since the default action is calling homeassistant.turn_on/turn_off I guess it's not implemented. 

I would really appreciate adding scene.turn_on for having some awesome scene tiles in my project.

Thank you!

**Comment by c727** [comment]
Version: 20180226
```
-added binary_sensor and scene support
-every domain can use more_info now
```

**Comment by andreasnanko** [comment]
***That*** was fast! 👍 
Will try, thank you!

**Comment by c727** [comment]
you are welcome, download the fixed version 20180226.1 :+1:
